src/main/resources/view/HelpPage.fxml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<VBox fx:id="help" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" prefHeight="540.0" prefWidth="1050.0" spacing="30.0" style="-fx-background-color: transparent;" stylesheets="@../styles/HelpPage.css" xmlns="http://javafx.com/javafx/16" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
maxHeight/maxWidth a bit funky but i'm just gonna assume that's stylebuilder shenanigans, so meh

also javafx 16?

Reply:
(and for all the other fxml files too i guess, re: javafx 16)

Reply:
ya that's MAX_INT

Reply:
this one still says javafx 16 btw